go/roothash/api/api.go

```diff
@@ -130,6 +138,25 @@ func NewRequestProposerTimeoutTx(nonce uint64, fee *transaction.Fee, runtimeID c
 	})
 }
 
+// RuntimeState is the per-runtime state.
+type RuntimeState struct {
+	Runtime   *registry.Runtime `json:"runtime"`
+	Suspended bool              `json:"suspended,omitempty"`
+
+	GenesisBlock *block.Block `json:"genesis_block"`
+
+	CurrentBlock       *block.Block `json:"current_block"`
+	CurrentBlockHeight int64        `json:"current_block_height"`
+
+	// LastNormalRound is the runtime round which was normally processed by the runtime. This is
+	// also the round that contains the message results for the last processed runtime messages.
+	LastNormalRound uint64 `json:"last_normal_round"`
```

Comment:
Maybe `LastNonEmpty{Round,Height}` would be more descriptive? Don't have any better suggestion, but i feel "Normal" is not the most descriptive.

Reply:
The reason why I used "Normal" is that this is the header type (`HeaderType`) of "normal" (:grin:) blocks. I agree that it is not the best term.

Reply:
Ah yeah that makes sense :-) Potentially could use `LastNormalBlock{Round,Height}`, but i'm also fine if it stays at it is.